tests/training/metrics/span_based_f1_measure_test.py

```diff
@@ -163,3 +172,65 @@ def test_span_f1_can_build_from_params(self):
         metric = Metric.from_params(params, self.vocab)
         assert metric._ignore_classes == ["V"]
         assert metric._label_vocabulary == self.vocab.get_index_to_token_vocabulary("tags")
+
+    def test_span_f1_matches_perl_script_for_continued_arguments(self):
+        bio_tags = ["B-ARG1", "O", "B-C-ARG1", "B-V", "B-ARGM-ADJ", "O"]
+        sentence = ["Mark", "and", "Matt", "were", "running", "fast", "."]
+
+        gold_indices = [self.vocab.get_token_index(x, "tags") for x in bio_tags]
+
+        gold_tensor = torch.Tensor([gold_indices])
+        prediction_tensor = torch.rand([1, 6, self.vocab.get_vocab_size("tags")])
+
+        # Test that the span measure ignores completely masked sequences by
+        # passing a mask with a fully masked row.
+        mask = torch.LongTensor([[1, 1, 1, 1, 1, 1, 1, 1, 1]])
+
+        # Make prediction so that it is exactly correct.
+        prediction_tensor[:, 0, 1] = 1
```

Comment:
Can you replace the `:` with `0`?  Also, it'd be nice to have some reference for the magic numbers in the third index, in a comment after each line, or something.  You could even make this so it's robust to future changes by doing something like:

```python
for i, tag_index in enumerate(gold_indices):
    prediction_tensor[0, i, tag_index] = 1
```

Fewer lines of code, even =).